docs/Tutorial_Consensus_Motif.ipynb

```diff
@@ -0,0 +1,610 @@
+{
```

Comment:
<p>I would avoid using the term "snippet" since, in <a href="https://www.cs.ucr.edu/~eamonn/Time_Series_Snippets_10pages.pdf" target="_blank">matrix profile parlance</a>, snippets has a very specific meaning and is overloaded. Could we please say "subsequences" instead?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/279/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>